From TravisCI on the Python 3.5.3 Pylint command:

`homeassistant/components/sensor/noaa_tides.py:115:8: E0401: Unable to import 'py_noaa' (import-error)`

--- a/homeassistant/components/sensor/noaa_tides.py
+++ b/homeassistant/components/sensor/noaa_tides.py
@@ -113,7 +113,7 @@ class NOAATidesAndCurrentsSensor(Entity):
 
     def update(self):
         """Get the latest data from NOAA Tides and Currents API."""
-        from py_noaa import coops # pylint: disable=import-error
+        from py_noaa import coops  # pylint: disable=import-error
         begin = datetime.now()
         delta = timedelta(days=2)
         end = begin + delta

--- a/homeassistant/components/sensor/noaa_tides.py
+++ b/homeassistant/components/sensor/noaa_tides.py
@@ -54,6 +54,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                                              timezone, unit_system)
     noaa_sensor.update()
     if noaa_sensor.data is None:
+        _LOGGER.error("Unable to setup NOAA Tides Sensor.")
         return
     add_devices([noaa_sensor], True)
 
@@ -112,7 +113,7 @@ class NOAATidesAndCurrentsSensor(Entity):
 
     def update(self):
         """Get the latest data from NOAA Tides and Currents API."""
-        from py_noaa import coops
+        from py_noaa import coops # pylint: disable=import-error
         begin = datetime.now()
         delta = timedelta(days=2)
         end = begin + delta

--- a/homeassistant/components/sensor/noaa_tides.py
+++ b/homeassistant/components/sensor/noaa_tides.py
@@ -14,7 +14,7 @@ from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import (ATTR_ATTRIBUTION, CONF_NAME, CONF_UNIT_SYSTEM)
 from homeassistant.helpers.entity import Entity
 
-REQUIREMENTS = ['py_noaa==0.2.2']
+REQUIREMENTS = ['py_noaa==0.3.0']
 
 _LOGGER = logging.getLogger(__name__)
 

